src/arrayReverse.test.js

```diff
@@ -1,20 +1,47 @@
-'use strict';
+"use strict";
 
 describe(`Function 'arrayReverse':`, () => {
-  const arrayReverse = require('./arrayReverse');
+  const arrayReverse = require("./arrayReverse");
 
   it(`should be declared`, () => {
     expect(arrayReverse).toBeInstanceOf(Function);
   });
 
-  it(`should return an array`, () => {
+  it(`should handle single-word arrays`, () => {
+    expect(arrayReverse(["Hello"])).toEqual(["olleH"]);
+  });
+
+  it(`should reverse the strings and their order in the array`, () => {
+    expect(arrayReverse(["Mate", "Academy"])).toEqual(["ymed", "acAetaM"]);
+  });
+
+  it(`should return an empty array for empty input`, () => {
+    expect(arrayReverse([])).toEqual([]);
+  });
 
+  it(`should handle arrays with empty strings`, () => {
+    expect(arrayReverse(["", "", ""])).toEqual(["", "", ""]);
   });
 
-  it(`should return an empty string
-    if original array consists of an empty string`, () => {
-    expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
+  it(`should reverse strings of mixed characters`, () => {
+    expect(arrayReverse(["I", "am", "a", "student!"])).toEqual([
+      "!",
+      "tn",
+      "e",
+      "dutsamaI",
+    ]);
   });
 
-  // write more tests here
+  it(`should return an array of strings of identical characters`, () => {
+    expect(arrayReverse(["aaa", "bbb"])).toEqual(["bbb", "aaa"]);
+  });
+
+  it(`should correctly reverse strings with numbers and symbols`, () => {
+    expect(arrayReverse(["123", "abc!"])).toEqual(["!cb", "a321"]);
```

Comment:
The expected output should reverse both the strings and their order in the array. The correct expected output should be `["!cba", "321"]`.